reactioncommerce/reaction
# [Marketplace] Fix currency on merchant shop to default to the primary shop currency.

Resolves #2668 

This bug was caused by the merchant shop picking the locale currency instead of shop currency. The dropdown currency was not changing the products currencies because it was using the `getShopId` to get the settings for the merchant shop instead of `getPrimaryShopId`. This PR makes sure that the default currency is set to the marketplace owner's shop's currency and that the currency dropdown to change currency picks the settings from the primary shop.

**How To Test** 

1. Use VPN or some other method so you are in a non-US locale
2. Invite a shop owner
3. Using incognito accept invitation
4. Create a product
5. Observe that product now displays in locale currency
6. Change dropdown, observe that currency does not change.
